bindings/python/contacts/contact-point.hpp

```diff
@@ -0,0 +1,158 @@
+#ifndef __tsid_python_contact_6d_hpp__
+#define __tsid_python_contact_6d_hpp__
+
+#include <boost/python.hpp>
+#include <string>
+#include <eigenpy/eigenpy.hpp>
+#include <boost/python/suite/indexing/vector_indexing_suite.hpp>
+
+//
```

Comment:
This header should appear at the top of the file and I think you should also add the credit of NYU.

Reply:
Thanks for pointing this out. I've move the comments to the first line and added "NYU, MPI Tübingen" to the copyright.